client/src/pages/Clusters/Clusters.tsx

```diff
@@ -99,49 +68,106 @@ const transformUpdatedAtDate = (cluster: ClusterSummary) => {
 };
 
 const Clusters = () => {
-  const [clusters, setClusters] = useState<ClusterDataRow[]>([]);
-  const [isLoading, setIsLoading] = useState(true);
-
-  const fetchClusters = async () => {
-    try {
-      const clustersData = await ManagmentServiceApiInstance.getClusters();
-
-      const clusterRows = clustersData.map(
-        (cluster): ClusterDataRow => ({
-          name: cluster.id,
-          accuracy: cluster.accuracy,
-          state: transformIsRunningLabel(cluster),
-          notificationChannels: transformNotificationChannelsToColumns(cluster),
-          updatedAt: transformUpdatedAtDate(cluster),
-        }),
-      );
-
-      setClusters(clusterRows);
-      setIsLoading(false);
-    } catch (e: unknown) {
-      console.error('Failed to fetch clusters', e);
-    }
-  };
-
-  useEffect(() => {
-    fetchClusters();
-  }, []);
-
-  const header = <HeaderWithIcon title={'Clusters'} />;
-
-  return (
-    <PageTemplate header={header}>
-      <SectionComponent title={'Clusters'} icon={<SVGIcon iconName='clusters-icon'/>}>
-        {isLoading && <div>Loading...</div>}
-        {!isLoading && clusters.length > 0 && (
-          <Table columns={columns} rows={clusters} />
-        )}
-        {!isLoading && clusters.length == 0 && (
-          <div>No registered clusters yet</div>
-        )}
-      </SectionComponent>
-    </PageTemplate>
-  );
+    const [clusters, setClusters] = useState<ClusterDataRow[]>([]);
```

Comment:
Consider reformatting these changes with prettier to avoid conflicts

Reply:
:D